lib/models/os.coffee

```diff
@@ -330,13 +344,22 @@ getOsModel = (deps, opts) ->
 	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
 	# });
 	###
-	exports.getConfig = (nameOrId, callback) ->
+	exports.getConfig = (nameOrId, options = {}, callback) ->
+		callback = findCallback(arguments)
+
+		defaultOpts =
+			appUpdatePollInterval: 60000
```

Comment:
Now that this is the same as resin-device-config's default, we should probably remove this too, and let that handle the defaults there itself.